config/install/field.field.user.user.field_gender.yml

```diff
@@ -14,7 +14,9 @@ label: Gender
 description: ''
 required: false
 translatable: false
-default_value: {  }
+default_value:
+  -
+    value: other
```

Comment:
Are you sure the default value should be "Other / Prefer not to say"?

Reply:
Zohar: "Other / Prefer not to say [Default]"

https://openideal-v3.atlassian.net/browse/OI-41?focusedCommentId=10146